tensorpack/train/trainers.py

```diff
@@ -370,10 +371,11 @@ class HorovodTrainer(SingleCostTrainer):
            for a full example which has handled these common issues.
            This example can train ImageNet in roughly an hour following the paper's setup.
     """
-    def __init__(self, average=True):
+    def __init__(self, average=True, compression=Compression.none):
```

Comment:
The default value should be `None` because `Compression` may not be imported when horovod is not installed.